### DISCUSSION TITLE
Quest 2 Support?

### DISCUSSION THREAD

**Discussion by Juris-S** [BioMotionLab/TUX#69] [Q&A] [answered]
Hey 👋,

I had a simple query if BMLTux works when building for Quest 2 in Unity, how does this translate with the requirement for a second display to populate the participant variables; can this process be circumvented?

Building on this, is there a way to switch out the experiment settings after the application is built, in other words, during run-time?

Thanks!

**Comment by AdamBebko** [comment, marked as answer]
Hi Juris, here's a small page on what other users have done to solve this:

https://biomotionlab.github.io/TUX/docs/StandaloneVr
